src/main/java/org/coffeemine/app/spring/components/RiskMatrix.java

```diff
@@ -0,0 +1,158 @@
+package org.coffeemine.app.spring.components;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.grid.editor.Editor;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.select.Select;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.data.binder.Binder;
+import org.coffeemine.app.spring.data.Risk;
+import org.coffeemine.app.spring.view.View;
+
+import java.util.*;
+
+public class RiskMatrix extends View {
+
+    private List<Risk> riskList;
+
+    public RiskMatrix(){
+
+        riskList = new ArrayList<>();
+        riskList.add(new Risk ("No or poor business case", "Commercial", "Select", "Select"));
```

Comment:
should be done more efficiently but cant blame ya for doing it like this

Reply:
Too bad Java doesn't have `constexpr`